packages/app-store/templates/audit-log-implementation/components/AuditLogEventToggles.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
```

Comment:
This component allows users to enable/disable events. This action itself will trigger a report, and this the only other event that can not/should not be disabled.